pycparser/c_lexer.py

```diff
@@ -100,24 +100,32 @@ def _make_tok_location(self, token):
     ## Reserved keywords
     ##
     keywords = (
-        '_BOOL', '_COMPLEX', 'AUTO', 'BREAK', 'CASE', 'CHAR', 'CONST',
+        '_BOOL', '_COMPLEX', '_IMAGINARY',
+        'AUTO', 'BREAK', 'CASE', 'CHAR', 'CONST',
         'CONTINUE', 'DEFAULT', 'DO', 'DOUBLE', 'ELSE', 'ENUM', 'EXTERN',
         'FLOAT', 'FOR', 'GOTO', 'IF', 'INLINE', 'INT', 'LONG',
         'REGISTER', 'OFFSETOF',
         'RESTRICT', 'RETURN', 'SHORT', 'SIGNED', 'SIZEOF', 'STATIC', 'STRUCT',
         'SWITCH', 'TYPEDEF', 'UNION', 'UNSIGNED', 'VOID',
         'VOLATILE', 'WHILE', '__INT128',
+        # C11
+        '_ALIGNOF', '_ATOMIC', '_THREAD_LOCAL', '_NORETURN',
```

Comment:
Can you add tests for all the new keywords? Lexer tests + parser tests would be best